exporters/stats/prometheus/src/main/java/io/opencensus/exporter/stats/prometheus/PrometheusExportUtils.java

```diff
@@ -88,127 +74,119 @@
-    List<String> labelNames = convertToLabelNames(view.getColumns());
+  // Converts a Metric to a Prometheus MetricFamilySamples.
+  static MetricFamilySamples createMetricFamilySamples(Metric metric) {
+    MetricDescriptor metricDescriptor = metric.getMetricDescriptor();
+    String name = Collector.sanitizeMetricName(metricDescriptor.getName());
+    Type type = getType(metricDescriptor.getType());
+    List<String> labelNames = convertToLabelNames(metricDescriptor.getLabelKeys());
     List<Sample> samples = Lists.newArrayList();
-    for (Entry<List</*@Nullable*/ TagValue>, AggregationData> entry :
-        viewData.getAggregationMap().entrySet()) {
-      samples.addAll(
-          getSamples(name, labelNames, entry.getKey(), entry.getValue(), view.getAggregation()));
+
+    for (io.opencensus.metrics.export.TimeSeries timeSeries : metric.getTimeSeriesList()) {
+      for (io.opencensus.metrics.export.Point point : timeSeries.getPoints()) {
+        samples.addAll(getSamples(name, labelNames, timeSeries.getLabelValues(), point.getValue()));
+      }
     }
-    return new MetricFamilySamples(name, type, view.getDescription(), samples);
+    return new MetricFamilySamples(name, type, metricDescriptor.getDescription(), samples);
   }
 
-  // Converts a View to a Prometheus MetricFamilySamples.
+  // Converts a MetricDescriptor to a Prometheus MetricFamilySamples.
   // Used only for Prometheus metric registry, should not contain any actual samples.
-  static MetricFamilySamples createDescribableMetricFamilySamples(View view) {
-    String name = Collector.sanitizeMetricName(view.getName().asString());
-    Type type = getType(view.getAggregation(), view.getWindow());
-    List<String> labelNames = convertToLabelNames(view.getColumns());
+  static MetricFamilySamples createDescribableMetricFamilySamples(
+      MetricDescriptor metricDescriptor) {
+    String name = Collector.sanitizeMetricName(metricDescriptor.getName());
+    Type type = getType(metricDescriptor.getType());
+    List<String> labelNames = convertToLabelNames(metricDescriptor.getLabelKeys());
     if (containsDisallowedLeLabelForHistogram(labelNames, type)) {
       throw new IllegalStateException(
           "Prometheus Histogram cannot have a label named 'le', "
               + "because it is a reserved label for bucket boundaries. "
               + "Please remove this tag key from your view.");
     }
     return new MetricFamilySamples(
-        name, type, view.getDescription(), Collections.<Sample>emptyList());
+        name, type, metricDescriptor.getDescription(), Collections.<Sample>emptyList());
   }
 
   @VisibleForTesting
-  static Type getType(Aggregation aggregation, View.AggregationWindow window) {
-    if (!(window instanceof View.AggregationWindow.Cumulative)) {
-      return Type.UNTYPED;
+  static Type getType(MetricDescriptor.Type type) {
+    if (type == MetricDescriptor.Type.CUMULATIVE_INT64
```

Comment:
`CUMULATIVE_INT64` and `CUMULATIVE_DOUBLE` can also represent cumulative sum, so the values could increment by more than 1 (e.g sum 10 -> 20 -> 30). Does Prometheus Counter allow that?

Reply:
FYI https://github.com/prometheus/client_java/blob/master/simpleclient/src/main/java/io/prometheus/client/Counter.java#L117-L126